apps/meteor/app/models/server/models/Subscriptions.js

```diff
@@ -30,7 +30,6 @@ export class Subscriptions extends Base {
 		this.tryEnsureIndex({ emailNotifications: 1 }, { sparse: 1 });
 		this.tryEnsureIndex({ autoTranslate: 1 }, { sparse: 1 });
 		this.tryEnsureIndex({ autoTranslateLanguage: 1 }, { sparse: 1 });
-		this.tryEnsureIndex({ 'userHighlights.0': 1 }, { sparse: 1 });
```

Comment:
still removing an index 🙈

Reply:
Wait, i removed that one lol. Thanks for fixing it.